themes-default/slim/views/vue-components/app-link.mako

```diff
@@ -13,6 +13,7 @@
 <script>
 Vue.component('app-link', {
     name: 'app-link',
```

Comment:
This line is redundant, please remove it.

Reply:
This is for future usage.

Reply:
Okay, then remove the one above it.
[It overrides the first argument to `Vue.component`.](https://vuejs.org/v2/api/#Vue-component)